src/Microsoft.PowerShell.Commands.Utility/commands/utility/AddType.cs

```diff
@@ -599,7 +599,7 @@ protected override void EndProcessing()
         // We now ship .Net Core's reference assemblies with PowerShell, so that Add-Type can work
```

Comment:
@daxian-dbw, your last commit had 4 failures in `PowerShell-CI-windows`
Tests Import-PSSession cmdlet works with types unavailable on the client.Verifies client-side unavailable enum is correctly handled
```powershell
Expected 2, but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Implicit.Remoting.Tests.ps1: line 130
130:                 (foo -x "Value2") | Should -Be 2
```
Import-PSSession with FormatAndTypes.Implicit remoting works even when types.ps1xml is missing on the client.Serialization works for top-level properties
```powershell
Expected exactly 'root', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Implicit.Remoting.Tests.ps1: line 702
702:                 $x.text | Should -BeExactly "root"
```
Import-PSSession with FormatAndTypes.Implicit remoting works even when types.ps1xml is missing on the client.Serialization settings works for deep properties
```powershell
Expected exactly 'Grandson', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Implicit.Remoting.Tests.ps1: line 707
707:                 $x.Son.Grandson.text | Should -BeExactly "Grandson"
```
Import-PSSession with FormatAndTypes.Implicit remoting works even when types.ps1xml is missing on the client.Serialization settings are preserved even if types.ps1xml is missing on the client
```powershell
Expected exactly 'Grandson', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\Implicit.Remoting.Tests.ps1: line 712
712:                 $y | Should -BeExactly "Grandson"
```